src/machines/authMachine.ts

```diff
@@ -146,9 +146,9 @@ async function getUser(input: { token?: string }) {
 
   /**
    * We do not want to store a token or a user since the developer is running
-   * the application locally. They know what they are doing.
+   * the application and dependencies locally. They know what they are doing.
    */
-  if (env().DEV && env().VITE_KITTYCAD_API_TOKEN === 'localhost') {
+  if (env().VITE_KITTYCAD_API_TOKEN === 'localhost') {
```

Comment:
This is still safe. We never set `VITE_KITTYCAD_API_TOKEN` in the production build.